docs/best-practices/continuous-integration/github.md

```diff
@@ -0,0 +1,50 @@
+# GitHub Actions Integration
+
+Use [GitHub Actions runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners) running on a macOS machine
+to build using fastlane.
+
+## Repository setup
+
+First create a `Gemfile` in the root of your project with the following content:
+
+```ruby
+source 'https://rubygems.org'
+
+gem 'fastlane'
+```
+
+Add a `.github/workflows/build-ios-app.yml` file to trigger _fastlane_.
```

Comment:
Let's give it a hint that `build-ios-app` is just a example name the user can modify 😃 

```suggestion
Add a workflow yaml file to your repository. For example, if naming your workflow "build-ios-app", add `.github/workflows/build-ios-app.yml` with the following content:
```